projects/kit/components/input-inline/input-inline.style.less

```diff
@@ -6,6 +6,14 @@
     overflow: hidden;
     white-space: nowrap;
     box-sizing: border-box;
+
+    &._focused .t-native {
```

Comment:
What are these changes for?

Reply:
For show native input element instead

Reply:
But that was this way on purpose, so it doesn't just by 0.5px when you focus since it's impossible to align input and div 100% the same